wiki/Contributing-to-the-Wiki.md

```diff
@@ -0,0 +1,67 @@
+We want to make sure that contributing to the Wiki is as easy as possible so that anyone that would like to help us with this task can do so. In that light, some guidelines to contribute to the Wiki are outlined below.
+
+## How the Wiki is updated
+Typically, a repository's Wiki can only be updated by the owners of that repo, this made community provided documentation cumbersome to include. In order to easily accept community provided documentation this repository is utilizing a [GitHub Action](https://github.com/Andrew-Chen-Wang/github-wiki-action) which will generate the Wiki from any Markdown files merged to the [`publish-wiki` branch's `/wiki/` folder](https://github.com/foundryvtt/dnd5e/tree/publish-wiki/wiki). If you would like to submit documentation to be included in the Wiki, please review the recommendations below and the steps to open an issue and submit a Pull Request below.
+
+## Style Guide
+When submitting a Markdown file for consideration for the Wiki, we ask you to follow the style guide outlined here, this will ensure consistency between all of our wiki pages.
+
+### Markdown Formatting
+Use [GitHub's Basic Formatting Syntax](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#headings) as a guide on how to format text with Markdown, as GitHub does not support all general Markdown formatting you may find in other guides.
+
+### Assets
+When taking a screenshot or screen recording, all modules should be disabled. The wiki is intended for core System functionality only.
+When adding an image or video, first upload the asset to the original Issue created for the update. This will provide us with a reliable link to that asset for use between your Fork and the dnd5e system.  
+- Images
+    - I dunno, any guidelines here?
+    - Recommended software for taking and annotating screenshots?
+- Videos
+    - Webm preferred? Does is matter?
+    - Recommended software to capture video?
+
+### File Names and Links
+The name of your Markdown file is what will be used for the name of the Wiki Page in the table of contents, with hyphens turning into spaces, so `My-first-Wiki-page.md` will become `My first Wiki page`. 
+When you want to link to another page in the wiki, in your file make sure to link to the Markdown file in the `wiki` folder, the GitHub Action will automatically strip the `.md` extension and generate the appropriate link to the wiki page, for example linking to `[Hyperlink Text](Advancement-Type-Hit-Points.md)` in your documents will always generate a link to `[Hyperlink Text](https://github.com/foundryvtt/dnd5e/wiki/Advancement-Type-Hit-Points)` in the Wiki.  
+Always make sure to update the `Home.md` file with a link to your new Markdown file.
+
+### That little version tag?
+***Do we want to have the version tag at the top of all of our docs to reflect the version the page was last updated?***  
```

Comment:
I think it's been pretty useful to have a badge to mention up to what version the information is accurate for. Whether we want to include the actual time the page was last updated I'm less concerned about, but it's generally useful to have that information somewhere.